quickstart/101-azure-api-management-create/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">=0.12"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>2.0"
```

Comment:
Should we use `"~>3.0"` instead?

Reply:
Yes. This is just the first checkin of an auto-generated sample. I haven't made any edits yet.

Reply:
Fixed